### PR TITLE
[PAN-1990] Read transfer data from destination blockchain

### PR DESCRIPTION
**Pull request by danut13**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Test(s)
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

Read transfer data from destination blockchain

### CODE REVIEW

File: pantos/client/library/blockchains/ethereum.py [hunk 1]
```diff
@@ -232,6 +268,11 @@ def _get_utilities(self) -> EthereumUtilities:
         # Docstring inherited
         return typing.cast(EthereumUtilities, super()._get_utilities())
 
+    def _create_unknown_transfer_error(
```
Comment (markuslevonyak) [inline pantos/client/library/blockchains/ethereum.py:271]: This function should be moved to `base.BlockchainClient`.

File: pantos/client/library/blockchains/ethereum.py [hunk 2]
```diff
@@ -192,6 +194,40 @@ def read_service_node_url(self,
...
+                from_block_number_ = max(to_block_number_ - blocks_per_query,
+                                         from_block_number)
+                transfer_event_logs = self._get_utilities().get_logs(
+                    transfer_event, from_block_number_, to_block_number_ - 1)
+                transfer_response = self.__find_destination_transfer(
+                    transfer_event_logs, request.source_transaction_id,
+                    request.source_blockchain, to_block_number)
+                if transfer_response:
+                    return transfer_response
+            raise self._create_unknown_transfer_error()
```
Comment (markuslevonyak) [inline pantos/client/library/blockchains/ethereum.py:224]: The `request` should be added here as a kwarg.

File: pantos/client/library/blockchains/ethereum.py [hunk 3]
```diff
@@ -246,3 +287,33 @@ def __sign_message(self, private_key: PrivateKey,
         signed_message = web3.Account.sign_message(message,
                                                    private_key=private_key)
         return signed_message.signature.to_0x_hex()
+
+    def __find_destination_transfer(
+            self, transfer_event_logs: typing.List[web3.types.EventData],
```
Comment (markuslevonyak) [inline pantos/client/library/blockchains/ethereum.py:292]: `list` instead of `typing.List`.

File: tests/conftest.py [hunk 4]
```diff
@@ -1,6 +1,7 @@
-"""Shared fixtures for all pantos.client.lbrary.business package tests.
+"""Shared fixtures for all pantos.client.lbrary package tests.
```
Comment (markuslevonyak) [inline tests/conftest.py:1]: `library` instead of `lbrary`.

**Review by markuslevonyak — APPROVED** [review]

**Review by chandraprakash — APPROVED** [review]

**Pull request merged**